pkg/api/install/install.go

```diff
@@ -59,15 +59,33 @@ func init() {
 
 		case *watchv1.Event:
 			switch b := objB.(type) {
+			case *watchv1.InternalEvent:
+				return true, watchv1.Convert_versioned_Event_to_versioned_InternalEvent(a, b, s)
 			case *watchapi.Event:
 				return true, watchv1.Convert_versioned_Event_to_watch_Event(a, b, s)
 			}
+		case *watchv1.InternalEvent:
+			switch b := objB.(type) {
+			case *watchv1.Event:
+				return true, watchv1.Convert_versioned_InternalEvent_to_versioned_Event(a, b, s)
+			}
 		case *watchapi.Event:
 			switch b := objB.(type) {
 			case *watchv1.Event:
 				return true, watchv1.Convert_watch_Event_to_versioned_Event(a, b, s)
 			}
 
+		case *kapi.ListOptions:
```

Comment:
Why didn't we get these from upstream?  They register the same types don't they?